src/network/netdev/batadv.c

```diff
@@ -133,6 +133,14 @@ static int netdev_batadv_post_create(NetDev *netdev, Link *link, sd_netlink_mess
         if (r < 0)
                 return log_netdev_error_errno(netdev, r, "Failed to set orig_interval: %m");
 
+        r = sd_netlink_message_append_u32(message, BATADV_ATTR_GW_BANDWIDTH_DOWN, b->gateway_bandwidth_down);
+        if (r < 0)
+                return log_netdev_error_errno(netdev, r, "Failed to set gateway_bandwidth_down: %m");
+
+        r = sd_netlink_message_append_u32(message, BATADV_ATTR_GW_BANDWIDTH_UP, b->gateway_bandwidth_up);
+        if (r < 0)
+                return log_netdev_error_errno(netdev, r, "Failed to set gateway_bandwidth_up: %m");
+
```

Comment:
All the commits
network: Add BridgeLoopAvoidance
network: Add DistributedArpTable
network: Add Fragmentation
network: Add HopPenalty and OrigInterval
network: Add GatewayBandwidthDown and GatewayBandwidthUp 
could be squashed together...